ios/MqttClient.swift

```diff
@@ -110,27 +110,48 @@ class MqttClient {
     }
 
     func subscribe(topic: String, qos: CocoaMQTTQoS) {
-        self.client.subscribe(topic, qos: qos)
+        do {
+            print("Subscribing to: \(topic) at qos: \(qos)")
+            self.client.subscribe(topic, qos: qos)
+        } catch {
+            print("Error: \(error)\nStack Symbols:\(Thread.callStackSymbols.forEach{ print($0) })")
```

Comment:
Rather than printing the error here, it should send the error back to the client via the error handler. So, something like this:

```
eventEmitter.dispatch(name: EventType.Error.rawValue, body: [
    "id": self.id,
    "cause": [ERROR_MESSAGE]
])
```

See the `mqttDidDisconnect` handler below for an example.